src/styles/blocks/container.scss

```diff
@@ -0,0 +1,21 @@
+.container {
+  display: grid;
+
+  justify-content: center;
+  gap: 46px 48px;
+  padding: 49px 40px;
+
+  grid-template-columns: repeat(auto-fit, 200px);
+
+  @media (min-width: 488px) {
+    grid-template-columns: repeat(auto-fit, 1fr);
+  }
+
+  @media (min-width: 768px) {
+    grid-template-columns: repeat(auto-fit, 1fr);
+  }
+
+  @media (min-width: 1024px) {
+    grid-template-columns: repeat(auto-fit, 1fr);
```

Comment:
The grid-template-columns property is repeated with the same value for different breakpoints. This is redundant and can be simplified. Consider combining these media queries into a single rule if there is no change in the value.